src/sentry/static/sentry/app/views/projectInstall/platform.jsx

```diff
@@ -142,13 +142,13 @@ const ProjectInstallPlatform = React.createClass({
             <div dangerouslySetInnerHTML={{__html: this.state.html}}/>
           )}
 
-          {queryParams.onboarding ?
+          {'signup' in queryParams ?
             <p>
-              <Link
-                to={`/${orgId}/${projectId}/?onboarding=1#welcome`}
+              <a
```

Comment:
Otherwise someone will come along and go, "this is dumb" and re-write it to use `<Link>`.